### PR TITLE
Run code coverage report on coveralls

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,4 +9,11 @@ branches:
...
     only:
         - master
-script: npm test
+matrix:
+  fast_finish: true
+  include:
+    - os: linux
+      env: CXX=g++-4.8 TEST_SUITE=test
+    - os: linux
+      env: CXX=g++-4.8 TEST_SUITE=coveralls
```
Comment (axic) [inline .travis.yml]: `CXX` flags aren't needed here

File: .travis.yml [hunk 2]
```diff
@@ -9,4 +9,11 @@ branches:
     # Pull request automation is not constrained to this set of branches.
     only:
         - master
-script: npm test
+matrix:
+  fast_finish: true
+  include:
+    - os: linux
+      env: CXX=g++-4.8 TEST_SUITE=test
```
Comment (axic) [inline .travis.yml]: This is not strictly needed as `pretest` includes it.

**Comment by axic** [comment]
Depends on #179 as node 4 build fails without that.

**Comment by chriseth** [comment]
istanbul? constantinople? byzantium? :)

**Comment by axic** [comment]
Should also add the icon to the README.

**Pull request merged**